### PR TITLE
Add warnings to fx for android articles

### PR DESCRIPTION
**Pull request by caitmuenster**
Add a warning note to articles about developing extensions for Firefox for Android about limited support in Fenix.

### CODE REVIEW

File: src/content/documentation/develop/developing-extensions-for-firefox-for-android.md [hunk 1]
```diff
@@ -5,15 +5,23 @@ permalink: /documentation/develop/developing-extensions-for-firefox-for-android/
...
-date: 2019-06-11 11:14:47
+date: 2019-08-04
 ---
 
 {% capture page_hero_banner_content %}
 
 # Developing extensions for Firefox for Android
 
+::: note alert
+In August 2020, Mozilla will begin rolling out a new, reimagined Firefox for Android experience. The  browser for Android has been rebuilt from the ground up using GeckoView, Mozilla's mobile browsing engine.
```
Comment (championshuttler) [inline src/content/documentation/develop/developing-extensions-for-firefox-for-android.md:18]: I think we can link `GeckoView` to https://mozilla.github.io/geckoview/ or something better if you know :)

File: src/content/documentation/develop/differences-between-desktop-and-android-extensions.md [hunk 2]
```diff
@@ -27,7 +27,11 @@ date: 2019-03-18
 # Differences between desktop and Android extensions
 
 ::: note alert
-**Warning: The content of this article may be out of date.** This page has last been updated before Firefox 54.
+In August 2020, Mozilla will begin rolling out a new, reimagined Firefox for Android experience. The  browser for Android has been rebuilt from the ground up using GeckoView, Mozilla's mobile browsing engine.
```
Comment (championshuttler) [inline src/content/documentation/develop/differences-between-desktop-and-android-extensions.md:30]: Same for this :)

File: src/content/documentation/develop/developing-extensions-for-firefox-for-android.md (hunk 1 above)
Comment (caitmuenster) [reply src/content/documentation/develop/developing-extensions-for-firefox-for-android.md:18]: Oh yeah good call! 👍

File: src/content/documentation/develop/developing-extensions-for-firefox-for-android.md [hunk 3]
```diff
@@ -5,15 +5,23 @@ permalink: /documentation/develop/developing-extensions-for-firefox-for-android/
...
+
+This article describes the development process of extensions on Firefox for Android through version 68. These methods are not supported for versions 69+.
+:::
+
 You'll approach the coding of an extension for Firefox for Android in the same way as you would for a desktop extension; using a text editor or tool of your choice to write the code. However, when you want to test and debug your extension you need to follow a different process, this article walks you through that process.
 
 Using [web-ext](/documentation/develop/getting-started-with-web-ext/) in your extension development is recommended. Follow the set up and debugging processes described here, but use [`web-ext run`](/documentation/develop/web-ext-command-reference#web-ext-run) to execute your extension on Firefox for Android. Among other advantages, using [web-ext](/documentation/develop/getting-started-with-web-ext/) automatically restarts your extension on Firefox for Android when you make edits. Also, you can take advantage of `web-ext lint` , which performs a check to determine if any of the permissions, manifest keys, and web extension APIs you’re using are incompatible with Firefox for Android.
```
Comment (muffinresearch) [inline src/content/documentation/develop/developing-extensions-for-firefox-for-android.md:27]: `web-ext lint` could also be a link.

Comment (caitmuenster) [reply src/content/documentation/develop/developing-extensions-for-firefox-for-android.md]: 👍 Added the link

**Pull request merged**